cmd/vic-machine/common/target.go

```diff
@@ -56,6 +58,16 @@ func (t *Target) TargetFlags() []cli.Flag {
 			Value: flags.NewOptionalString(&t.Password),
 			Usage: "ESX or vCenter password",
 		},
+		cli.BoolTFlag{ // TODO: s/BoolTFlag/BoolFlag/ - should be false by default
+			Name:        "insecure",
```

Comment:
drat, must have added it in my PR.
https://github.com/vmware/vic/pull/2553/files#diff-5a99f2371e79e7b3ef78eefdc3419581R443